src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -176,17 +189,79 @@ export class HaIntegrationCard extends LitElement {
     `;
   }
 
+  private _logsLink() {
+    return html`<a href="/config/logs"
+      >${this.hass.localize(
+        "ui.panel.config.integrations.config_entry.logs"
+      )}</a
+    >`;
+  }
+
   private _renderSingleEntry(item: ConfigEntryExtended): TemplateResult {
     const devices = this._getDevices(item);
     const services = this._getServices(item);
     const entities = this._getEntities(item);
 
+    let header: [string, ...unknown[]] | undefined;
+
+    if (item.disabled_by) {
+      header = [
+        "ui.panel.config.integrations.config_entry.disable.disabled_cause",
+        "cause",
+        this.hass.localize(
+          `ui.panel.config.integrations.config_entry.disable.disabled_by.${item.disabled_by}`
+        ) || item.disabled_by,
+      ];
+    } else if (item.state === "not_loaded") {
+      header = ["ui.panel.config.integrations.config_entry.not_loaded"];
+    } else if (item.state === "setup_retry") {
+      header = [
+        "ui.panel.config.integrations.config_entry.is_setup_retry",
+        "logs_link",
+        this._logsLink(),
+      ];
+    } else if (
+      item.state === "setup_error" ||
+      item.state === "migration_error"
+    ) {
+      header = [
+        "ui.panel.config.integrations.config_entry.is_setup_error",
+        "logs_link",
+        this._logsLink(),
+      ];
+    } else if (item.state === "failed_unload") {
+      header = [
+        "ui.panel.config.integrations.config_entry.is_failed_unload",
+        "logs_link",
+        this._logsLink(),
+      ];
+    }
+
+    const icons: [string, string][] = [];
+
+    if (this.manifest && !this.manifest.is_built_in) {
+      icons.push([
+        mdiPackageVariant,
+        this.hass.localize(
+          "ui.panel.config.integrations.config_entry.provided_by_custom_component"
+        ),
+      ]);
+    }
+    if (item.connection_class.startsWith("cloud_")) {
```

Comment:
Does it make sense to use the icons here https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things/#classifiers ?

Reply:
I only wanted to show icons if it's something that might cause issues (custom, cloud). I don't want to use that space to highlight every integration connection class, because then having an icon there no longer means "something bad"